build-support/bin/check_banned_imports.py

```diff
@@ -11,20 +11,7 @@
 
 def main() -> None:
   rust_files = find_files("src/rust/engine", extension=".rs")
-  python_files = find_files(
-    "src", "tests", "pants-plugins", "contrib", extension=".py"
-  )
```

Comment:
Is this a leftover? Not sure why it's going away.

Reply:
This is going away because the import we banned was the normal python `from enum import Enum`, which is now correct since we've removed `from pants.util.collections import Enum`!